src/Channels/SlackWebhookChannel.php

```diff
@@ -37,22 +37,32 @@ public function __construct(HttpClient $http)
      */
     public function send($notifiable, Notification $notification)
     {
-        if (! $url = $notifiable->routeNotificationFor('slack', $notification)) {
+        if (! $route = $notifiable->routeNotificationFor('slack', $notification)) {
             return;
         }
 
+        if (substr($route, 0, 4 ) === 'xoxp') {
+            $url = 'https://slack.com/api/chat.postMessage';
+            $token = $route;
+        } else {
+            $url = $route;
+            $token = null;
+        }
+
         return $this->http->post($url, $this->buildJsonPayload(
-            $notification->toSlack($notifiable)
+            $notification->toSlack($notifiable),
+            $token
         ));
     }
 
     /**
      * Build up a JSON payload for the Slack webhook.
      *
+     * @param  string  $token
      * @param  \Illuminate\Notifications\Messages\SlackMessage  $message
      * @return array
      */
-    protected function buildJsonPayload(SlackMessage $message)
+    protected function buildJsonPayload(SlackMessage $message, string $token=null)
```

Comment:
This is unfortunately a breaking change for anyone overriding the class.

Reply:
Thanks for the quick reply. I now slightly changed the code to avoid the breaking change you pointed out.